lib/konfa/deprecation.rb

```diff
@@ -0,0 +1,23 @@
+module Konfa
```

Comment:
Need to add tests for this too

Reply:
https://github.com/promoteinternational/konfa/blob/f755be3b5c1a239e899bd520b6b18a081969060c/spec/konfa/deprecation_spec.rb?

Reply:
Sorry, didn't see that. Excellent!